### PR TITLE
Djg 11 exercises with templates

### PR DESCRIPTION
**Pull request by PetroLakhai**
I worked here about:
- Updated the population script
- Modified the index page to also included the top five most viewed pages
- Included a link back to the index page from the category page

### CODE REVIEW

File: rango/models.py [hunk 1]
```diff
@@ -1,18 +1,35 @@
...
+
+    class Meta:
+        verbose_name_plural = 'categories'
 
     def __str__(self):
         return self.name
 
 
 class Page(models.Model):
+    list_display = ("title", "category", "url")
```
Comment (GrytsenkoDaria) [inline rango/models.py]: What do we need this code for?

File: templates/rango/category.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <title>Rango</title>
+    </head>
+
+    <body>
+        <div>
+        {% if category %}
+            <h1>{{ category.name }}</h1>
+            {% if page %}
+            <ul>
+                {% for pag in page %}
```
Comment (GrytsenkoDaria) [inline templates/rango/category.html]: Typo, there should be "page" not "pag"

**Review by GrytsenkoDaria — CHANGES_REQUESTED** [review]

File: rango/models.py (hunk 1 above)
Comment (PetroLakhai) [reply rango/models.py]: It needs to show three columns "title", "category", "url" on the admin page. I understood the mistake the row №23 I deleted here in model.py and added the row to admin.py.

File: templates/rango/category.html (hunk 2 above)
Comment (PetroLakhai) [reply templates/rango/category.html]: Ok, understood the mistake here - not correct naming. I changed rows as you proposed.

**Review by GrytsenkoDaria — APPROVED** [review]

**Pull request merged**